capabilities/combining-capabilities.md

```diff
@@ -76,3 +76,24 @@ For `trn` and `ref` we need to return a locally immutable reference capability t
 ### Reading from a `tag` variable
 
 This one is easy: `tag` variables are opaque! They can't be read from.
+
+## Writing to the field of an object
+
+Like reading the field of an object, writing to a field depends on the reference capability of the object reference being stored and the reference capability of the origin object containing the field. The reference capability of the object being stored must not violate the guarantees made by the origin object's reference capability. For example, a val object reference can be stored in an iso origin. This is because the tag reference capability guarantees that no alias to that object exists which could violate the guarantees that the iso capability makes.
```

Comment:
Is this a typo?

`This is because the tag reference capability guarantees that no alias...` Should this be `val`? If not How do we get from a `val` field on an `iso` to a `tag`?

Reply:
Yes, it should. Thanks!